docs/en/05_Contributing/09_Core_committers.md

```diff
@@ -55,3 +70,23 @@ often working alongside Core Committers. They are guided by additional rules:
  * For these complex or impactful changes, Core Committers should be given 1-2 working days to review. Ideally at this point, the API has already been agreed on through issue comments outlining the planned work (see [RFC Process](request_for_comment]).
  * More straightforward changes (e.g. documentation, styling) or areas which require quite specialised expertise (e.g. React) that's less available through most Core Committers can be approved or merged by team members who aren't Core Committers
  * Self-merges should be avoided, but are preferable to having work go stale or forcing other team members to waste time by context switching into a complex review (e.g. because the original reviewer went on leave). Any self-merge should be accompanied by a comment why this couldn't be handled in another way, and a (preferably written) approval from another team member.
+
+This role may be granted by any Core Committer,
+who should give other Core Committers an opportunity to weigh in on the decision.
+
+### Triage
+
+Triage of issues and pull request is an important activity: Reviewing issues, adding labels,
+closing stale issues, etc. This does not require write access to the repository as a "Contributing Committer".
+This is a great way for active community members to help out, and start a path towards becoming a Core Committer.
+
+Triage roles may be granted by any Core Committer,
+who should give other Core Committers an opportunity to weigh in on the decision.
+## Supported Modules
```

Comment:
Blank line above?

Reply:
Hah thanks Michal, started using VSCode and it has this thing where it shows the `git blame` in a "fake line" above, which makes it easy to miss when there's no actual blank line. Fixed now.